documentation/latest/intro/overview.md

```diff
@@ -3,34 +3,43 @@ layout: documentation
-unknown printer took a galley of type and scrambled it to make a type specimen
-book. It has survived not only five centuries, but also the leap into electronic
-typesetting, remaining essentially unchanged. It was popularised in the 1960s
-with the release of Letraset sheets containing Lorem Ipsum passages, and more
-recently with desktop publishing software like Aldus PageMaker including
-versions of Lorem Ipsum.
-
-## Phase 2
-
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
-Ipsum has been the industry's standard dummy text ever since the 1500s, when an
-unknown printer took a galley of type and scrambled it to make a type specimen
-book. It has survived not only five centuries, but also the leap into electronic
-typesetting, remaining essentially unchanged. It was popularised in the 1960s
-with the release of Letraset sheets containing Lorem Ipsum passages, and more
-recently with desktop publishing software like Aldus PageMaker including
-versions of Lorem Ipsum.
-## Phase 3
-
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
-Ipsum has been the industry's standard dummy text ever since the 1500s, when an
-unknown printer took a galley of type and scrambled it to make a type specimen
-book. It has survived not only five centuries, but also the leap into electronic
-typesetting, remaining essentially unchanged. It was popularised in the 1960s
-with the release of Letraset sheets containing Lorem Ipsum passages, and more
-recently with desktop publishing software like Aldus PageMaker including
-versions of Lorem Ipsum.
+## Fleet management for edgedevices
+
+Flotta adds a way to manage all your fleet of devices from a single point, in
+this case, using Kubernetes CRD to handle all kinds of device configurations.
+
+Each Edgedevice can register, pull configuration using the Fleet Management API
+and keep syncing the data using a secure MTLS protocol between the device and
+the Fleet Management API.
+
+At the same time, you can schedule a container workload to run in the device
+using a Workload CRD, where a container workload can be scheduled for an
+unlimited number of devices.
+
+### Fleet observability
+
+Running container workloads in the edge device is the first part, but at the
+same time, the Device control plane takes care of the following actions in the
+workload to make edge-administration life easier.
+
+  - Get metrics from workloads and push to a remote serve when the connectivity is
+    up.
+  - Get logs from workloads and push to a remote server when the connection is up.
+
+### Device configuration
+
+Device configuration is a hard part on edge, this is why Flotta trusts
+RPM-Ostree technology to run updates on the non containerized workloads.
+
+At the same time, each device can be configured at a single unit - where all
+observability, logs, heartbeats can be set- or can be defined on DeviceSets. This
```

Comment:
I would suggest to remove "Add the same time,"
Should "configured at a single unit" be "configured **as** a single unit" ?
s/`set-`/`set -`/ (missing whitespace after set)